contracts/StablecoinConverter.sol

```diff
@@ -42,25 +46,52 @@ contract StablecoinConverter is EpochTokenLocker {
         uint128 remainingAmount; // remainingAmount can either be a sellAmount or buyAmount, depending on the flag isSellOrder
     }
 
+    struct PreviousSolutionData {
+        uint32 batchId;
+        TradeData[] trades;
+        uint16[] tokenIdsForPrice;
+        address solutionSubmitter;
+        uint256 feeReward;
+        uint256 objectiveValue;
+    }
+
+    struct TradeData {
+        address owner;
+        uint128 volume;
+        uint16 orderId;
+    }
+
     // User-> Order
```

Comment:
Public members should also have docs IMO as they generate public view functions.